packages/gafl-webapp-service/src/locales/cy.json

```diff
@@ -121,21 +121,21 @@
   "check_conf_contact_correct": "Mae’n gywir",
   "check_conf_contact_title_1": "A yw’r ",
   "check_conf_contact_title_2": " yn gywir?",
-  "client_error_content_if_not_working": "If that does not work, you will have to ",
-  "client_error_content_msg": "Sorry, there is a problem with the service",
-  "client_error_content_sorry": ". Sorry.",
-  "client_error_content_start_again": "start again",
+  "client_error_content_if_not_working": "Os nad yw hynny’n gweithio, bydd angen i chi ",
+  "client_error_content_msg": "Mae'n ddrwg gennym ond mae problem gyda’r gwasanaeth",
+  "client_error_content_sorry": ". Ymddiheuriadau.",
+  "client_error_content_start_again": "a dechrau eto",
   "client_error_content_try_another": "try to get back to your application",
```

Comment:
I think we need a translation for this too - particularly given that the first part of the sentence will no longer be in English, so we'd have "Gallwch try to get back to your application"